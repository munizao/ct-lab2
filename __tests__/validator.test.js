const {
  Validator
} = require('../lib/validator.js');

describe ('validator class module', () => {
  const testValidator = new Validator('name', {
    type: String,
    required: true
  });
  it('properly makes a validator', () => {
    expect(testValidator.type).toBeDefined();
    expect(testValidator.required).toBeDefined();
    expect(testValidator.validate).toBeDefined();
  });
});