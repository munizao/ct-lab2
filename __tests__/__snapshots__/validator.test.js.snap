// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`validator class module properly validates 1`] = `"received is not a function"`;

exports[`validator class module properly validates 2`] = `"received is not a function"`;
